pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -396,6 +396,11 @@ func createNodeCRD(clientset apiextensionsclient.Interface) error {
 											Description: "min-allocate is the minimum number of IPs that will be allocated before" +
 												" the cilium-agent will write the CNI config.",
 										},
+										"max-allocate": {
+											Type: "integer",
+											Description: "max-allocate is the maximum number of IPs that will be allocated to the" +
+												" node.",
```

Comment:
Nit: Some lines are already longer than this in the file, so it's maybe not worth wrapping for just one word? (Here and on the other occurrence below)

[Sorry, somehow I failed to add this comment to the previous review.]

Reply:
indeed 🤦‍♂ , 👍